webapp/src/components/properties/lastModifiedBy/lastModifiedBy.tsx

```diff
@@ -32,8 +30,12 @@ const LastModifiedBy = (props: Props): JSX.Element => {
     }
 
     return (
-        <div className='LastModifiedBy octo-propertyvalue readonly'>
-            {(boardUsersById && boardUsersById[latestBlock.modifiedBy]?.username) || latestBlock.modifiedBy}
+        <div>
```

Comment:
Is there any reason to have a container `div` here? Looks like not needed unless you want to add something like `LastModifiedBy` class. Actually, you are importing the `lastModifiedBy.scss` but you are not using it, because the class is no longer available.